pyintelowl/pyintelowl.py

```diff
@@ -167,8 +188,8 @@ def send_file_analysis_request(
 
     def send_observable_analysis_request(
         self,
-        analyzers_requested: List[str],
         observable_name: str,
+        analyzers_requested: List[str] = [],
```

Comment:
can u move this back to after `self` ? we shouldn't change the positions of arguments because that can break the code for a lot of people.